lib/screens/dateTimePicker.js

```diff
@@ -0,0 +1,44 @@
+import React, { useState } from 'react';
+import { StyleSheet, View, Button } from 'react-native';
+import DateTimePickerModal from "react-native-modal-datetime-picker";
+
+export default DateTimePicker = () => {
+    const [date, setDate] = useState(new Date());
+    const [isVisible, setVisible] = useState(false);
+
+    const pressPicker = () => {
+        setVisible(true);
+    };
+
+    const hidePicker = () => {
+        setVisible(false);
+    };
+
+    const onChange = selectedDate => {
+        setVisible(false);
+        setDate(selectedDate || date);
+    };
+
+    return (
+        <View style={styles.dateTimePickerBoxContainer}>
+            <Button
+                title={date.toLocaleString([], {year: '2-digit', month: '2-digit', day: '2-digit', hour: '2-digit', minute: '2-digit'})}
```

Comment:
This is long and ugly, might consider putting this into a const and sending the const into the prop instead